cypress/integration/volunteer/generate_court_report.spec.js

```diff
@@ -8,10 +8,16 @@ context("Logging into cypress as a volunteer", () => {
   it("should generate a court report", () => {
     cy.get("#toggle-sidebar-js").click();
     cy.contains("Generate Court Reports").click();
-    cy.get("#case-selection").select("CINA-18-1003 - non-transition");
```

Comment:
The changes in this test are not related. But I didn't realize that this test that I wrote yesterday was failing on master. I have fixed it by removing the static string so it can select whatever case is in the dropdown.